google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -999,6 +1009,17 @@ public MetricsProvider getMetricsProvider() {
       return this.metricsProvider;
     }
 
+    /** Set the endpoint for publishing client side metrics. */
+    public Builder setMetricsEndpoint(String endpoint) {
```

Comment:
This needs a bit more documentation and be more explicit that this is for the builtin metrics